code/game/objects/structures/girders.dm

```diff
@@ -161,7 +172,25 @@
 				return ..()
 
 	else
```

Comment:
```suggestion
	else if(W.force)
```

Maybe add this if you don't want people going through the whole `do_after` and other processing, and making a smash sound effect when hitting a wall with a cuddly toy?